content/id/docs/concepts/architecture/controller.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Controllers
+content_template: templates/concept
+weight: 30
+---
+
+{{% capture overview %}}
+
+Dalam bidang robotika dan otomatisasi, _control loop_ adalah
+lingkaran tertutup yang mengatur keadaan suatu sistem.
+
+Berikut adalah salah satu contoh loop kontrol: termostat di sebuah ruangan.
```

Comment:
need consistency: _control loop_ or _loop_ kontrol

Reply:
I think we need to change into "kontrol tertutup", but I will give both in the first-mentioned since it is a new term.